builder/bin/image_gen.rs

```diff
@@ -25,6 +30,10 @@ fn main() {
         )
         .arg(arg!(--"fake-rom" [FILE] "Fake ROM").value_parser(value_parser!(PathBuf)))
         .arg(arg!(--"fake-fw" [FILE] "Fake FW bundle image").value_parser(value_parser!(PathBuf)))
+        .arg(
+            arg!(--"hashes" [FILE] "JSON file with image bundle header hashes for external signing tools")
+                .value_parser(value_parser!(PathBuf)),
```

Comment:
The current help string doesn't make it clear to me that this is an "out" parameter

I can't think of a great way to phrase this but here is an attempt
```suggestion
            arg!(--"hashes" [FILE] "File path for output JSON file containing image bundle header hashes for external signing tools")
                .value_parser(value_parser!(PathBuf)),
```

Reply:
I like your wording better. Thanks!